test/Driver/toolchain_darwin.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %swiftc_driver -v -tools-directory %S/Inputs/fake-toolchain %s 2>&1 | %FileCheck -check-prefix LDSUB %s
+
+// REQUIRES: objc_interop
```

Comment:
Actually, since we pick based on target rather than host, just provide an explicit target here.